packages/benchmark/src/n8nApiClient/n8nApiClient.ts

```diff
@@ -0,0 +1,78 @@
+import axios, { AxiosError, AxiosRequestConfig } from 'axios';
```

Comment:
Any reason not to use fetch? It is included out of the box since node` v18.20.4`?

Reply:
I was first going with that approach but the API is a bit more cumbersome than with axios and since we already use axios it's unified that way